spec/models/vacancy_spec.rb

```diff
@@ -186,6 +186,41 @@
           .to eq(['must not be more than £200000'])
         expect(job.errors.messages[:maximum_salary]).to be_empty
       end
+
+      it 'can accept salary format with comma' do
+        job = build(:vacancy, minimum_salary: '20,000')
+
+        expect(job.valid?).to be true
+        expect(job.minimum_salary).to eq '20000'
+      end
+
+      it 'can tolerate salary format with decimal points at the end' do
+        job = build(:vacancy, minimum_salary: '20,000.')
+
+        expect(job.valid?).to be true
+        expect(job.minimum_salary).to eq '20000'
+      end
+
+      it 'can accept salary format with one decimal point' do
+        job = build(:vacancy, minimum_salary: '25000.4')
+        expect(job.valid?).to be true
+        expect(job.minimum_salary).to eq '25000.4'
+      end
```

Comment:
this seems to go against the acceptance criteria of the ticket. I think that this should be rejected based on the acceptance criteria

Reply:
Oh, in that case, the behaviour was correct before this test!